src/cljs/org/broadinstitute/firecloud_ui/page/workspace/analysis/track_selector.cljs

```diff
@@ -84,10 +84,11 @@
                             (style/right-ellipses {} "Please ensure you have the associated .bai or .bam.bai at the same path")]
                            [:div {:style {:flex "1 1 auto" :marginLeft 8 :overflow "hidden"}}
                             (style/left-ellipses {} track-url)
-                            (style/right-ellipses {} "Optional index file not found.")])
+                            (style/right-ellipses {:style {:fontStyle "italic"}} "Optional index file not found.")])
                   [:div {:style {:flex "1 1 auto" :marginLeft 8 :overflow "hidden"}}
                    (style/left-ellipses {} track-url)
-                   (style/left-ellipses {} @index-url)])
+                   ; the following padding because overflow is cutting off italicized text
```

Comment:
Note: I believe line comments start with a double ;; in Clojure style conventions. Don't worry about fixing, just an FYI.